index.html

```diff
@@ -8,20 +8,68 @@
   <title>Awesome Books</title>
 </head>
 <body>
-  <div id="form">
-  <h1>Awesome books</h1>
-	<div class="books"></div>
-  <hr>
-  <div  id="new_book"><h1>Add new book</h1></div>
+  <header>
+    <a id="logo" href="#">Awesome books</a>
+    <ul>
+      <li>
+        <a id = "list" href="#">List</a>
+      </li>
+            <li>
+        <a id = "New" href="#">Add new</a>
+      </li>      
+      <li>
+        <a id="cont" href="#">Contact</a>
+      </li>      
+    </ul>
```

Comment:
Make one of the menus to be active after being clicked instead wrapping it with a color, kindly make it an active instead of wrapping it. Check the below image to see how hard it is to find the List menu

![rr18](https://user-images.githubusercontent.com/55635977/134768610-95cbc190-779f-4f77-be42-a0c0142fb402.png)